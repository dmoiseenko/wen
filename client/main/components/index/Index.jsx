import React from 'react';
import { Route } from 'react-router-dom';

import TopBar from '../topbar/TopBar';
import Home from '../pages/HomePage';
import Notes from '../pages/NotesPage';
import About from '../pages/AboutPage';
import * as routes from '../../../../common/routes';
import Graph from '../pages/GraphPage';
import NotificationsContainer from '../notification/Notifications.container';


export default function Index() {
  return (
    <div className="index">
      <nav className="nav nav__topbar">
        <TopBar />
      </nav>
      <section className="section section__content">
        <Route exact path={routes.home} component={Home} />
        <Route path={routes.about} component={About} />
        <Route path={routes.notes} component={Notes} />
        <Route path={routes.graph} component={Graph} />
      </section>
      <NotificationsContainer />
    </div>
  );
}
