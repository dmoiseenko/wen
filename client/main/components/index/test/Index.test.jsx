import React from 'react';
import { shallow } from 'enzyme';

import Index from '../Index';


it('should render', () => {
  const wrapper = shallow(
    <Index />
  );

  expect(wrapper.node).toMatchSnapshot();
});
