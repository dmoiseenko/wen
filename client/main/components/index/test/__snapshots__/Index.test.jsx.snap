// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="index"
>
  <nav
    className="nav nav__topbar"
  >
    <TopBar />
  </nav>
  <section
    className="section section__content"
  >
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/about"
    />
    <Route
      component={[Function]}
      path="/notes"
    />
    <Route
      component={[Function]}
      path="/graph"
    />
  </section>
  <Connect(Notifications) />
</div>
`;
